tslint.json

```diff
@@ -2,34 +2,111 @@
   "rules": {
```

Comment:
Does it make sense to use the tslint recommended and overwrite only the rules we like to have different?

E.g. use-isnan

Reply:
Good point, done